ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -445,6 +445,11 @@ public void testEmptyInput() throws Exception {
     localCli.runNonInteractively("");
   }
 
+  @Test
+  public void testContinuation() throws Exception{
+    localCli.runNonInteractively("SHOW" + "\\\t \t\n"  + "TABLES;");
```

Comment:
Shouldn't this test case be simply `SHOW\nTABLES` to reproduce the original bug?

Reply:
@apurvam ready 👍